helfomat-web/src/main/resources/logback-spring.xml

```diff
@@ -14,18 +16,9 @@
         </rollingPolicy>
     </appender>
 
-    <springProfile name="console-log">
-        <include resource="org/springframework/boot/logging/logback/console-appender.xml"/>
-        <root level="ERROR">
-            <appender-ref ref="CONSOLE"/>
-            <appender-ref ref="ROLLING-FILE"/>
-        </root>
-    </springProfile>
-
-    <springProfile name="!console-log">
-        <root level="ERROR">
-            <appender-ref ref="ROLLING-FILE"/>
-        </root>
-    </springProfile>
+    <root level="ERROR">
+        <appender-ref ref="CONSOLE"/>
+        <appender-ref ref="ROLLING-FILE"/>
```

Comment:
can we delete the entire file?

Reply:
Since we only need the console output going forward, yes.
Removed the file and adjusted root log level in application.yml